Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/FareParameters.hs

```diff
@@ -48,6 +51,7 @@ data FareParametersDetails = ProgressiveDetails FParamsProgressiveDetails | Slab
 data FParamsProgressiveDetails = FParamsProgressiveDetails
   { deadKmFare :: HighPrecMoney,
     extraKmFare :: Maybe HighPrecMoney,
+    rideDurationFare :: Maybe HighPrecMoney,
```

Comment:
lets name it timeBasedFare.

Reply:
There is conflict with existing names,`timeBasedFare` and `timeFare` is used. 
To fix this will have to move those types to their specific FareParams types and refactor imports everywhere.
Should I do it or if possible we can have separate variable names ?